deno.jsonc

```diff
@@ -1,4 +1,11 @@
 {
+    "tasks": {
+        "setup": "npm install && deno cache --check src/mod.ts",
+        "backport": "npm run backport",
+        "test": "deno test --seed=123456 ./test/",
+        "coverage": "deno task test --coverage=./test/cov_profile && deno coverage --lcov --output=./coverage.lcov ./test/cov_profile",
+        "bundle": "cd bundling && ./bundle-all.sh"
```

Comment:
Some of these tasks are duplicated in `package.json`. Maybe they should be removed either here or there.

Reply:
Thanks, fixed it up.

Reply:
I guess he means you are duplciating scripts in package.json and deno.jsonc. It should be only in one place. So, you are not fixed the take.

Reply:
Well, the logic is not duplicated. Having npm scripts for everything is mostly a convenience thing for Node users. Should we remove all npm scripts completely?